lxd/containers_post.go

```diff
@@ -201,117 +201,104 @@ func createFromNone(d *Daemon, req *containerPostReq) Response {
 }
 
 func createFromMigration(d *Daemon, req *containerPostReq) Response {
-	if req.Source.Mode != "pull" {
-		return NotImplemented
+	push := false
```

Comment:
Does `Mode` need to be fully validated to prevent undesired behavior on a typo?  Also, moving the declaration of `push` down to where its used might help readability, though separating the check of `req.Source.Mode` from its consumption might be a little weird.

E.g.

```go
	if req.Source.Mode != "pull" && req.Source.Mode != "push" {
		return NotImplemented
	}

	...

	push := false
	if req.Source.Mode == "push" {
		push = true
	}

	migrationArgs := MigrationSinkArgs{
	...
		Push:	push,
	}
```